src/db/entities/BaseEntity.ts

```diff
@@ -21,8 +23,8 @@ export abstract class BaseEntity {
   @VersionColumn()
   version!: number;
 
-  @Column({ type: 'varchar', length: 255, nullable: true })
-  appInstanceId!: string | null;
+  @Column({ type: 'varchar', nullable: true })
```

Comment:
```suggestion
  @Column({ type: 'varchar' })
```

and we probably have an upper boundary for the length of this string?